examples/draw-geometry.html

```diff
@@ -0,0 +1,9 @@
+---
+layout: example.html
```

Comment:
as for the example name, maybe render-geometry might be a bit more self explaining? Draw geometry sounds more like people drawing a geometry with draw tools. Render geometry more captures the feeling of the GeoExt legend (which was called FeatureRenderer IIRC).